apps/backend/test/unit/reconciliation/helpers.ts

```diff
@@ -0,0 +1,79 @@
+import { faker } from '@faker-js/faker';
+import { differenceInMinutes, format, getTime, parse } from 'date-fns';
+import { MatchStatus } from '../../../src/common/const';
+import { POSDepositEntity } from '../../../src/deposits/entities/pos-deposit.entity';
+import { PaymentEntity } from '../../../src/transaction/entities';
+import { paymentMethods } from '../../../test/mocks/const/payment-methods';
+import { locations } from '../../mocks/const/locations';
+import { MockData } from '../../mocks/mocks';
+
+export const setSomePaymentsToTwentyMinutesLater = (
```

Comment:
This is also "resetting" the payments right (as in setting the status to pending)?

I'd suggest just passing an array into this function and calling it something like `resetPaymentsDataset` or something, and doing the 20 minute thing for each even index within the function rather than passing in an index. With some comments to explain why.

Reply:
The payments are reset between tests using:
(this is a recent change :) )

```
  afterEach(() => {
    jest.clearAllMocks();
  });
```

Function has been updated to receive an array, and status updates are removed (it was _redundant_ since the mock data generates the status as PENDING and each test will generate new data)

:)

Reply:
sorry for the caps-lock on PENDING, I wasn't yelling... that's how it is written in the codebase (as an enum)